docs/sql-server/sql-server-2012-release-notes.md

```diff
@@ -99,8 +101,9 @@ You can install SQL Server on Windows Server 2008 R2 Server Core SP1, with the f
   
 If these tasks are not completed, you will see the following error message when you try to create a semantic index.  
   
-<pre>Msg 41209, Level 16, State 3, Line 1  
-A semantic language statistics database is not registered. Full-text indexes using 'STATISTICAL_SEMANTICS' cannot be created or populated.</pre>  
+```Msg 41209, Level 16, State 3, Line 1  
```

Comment:
We have to stick to regular code block values here or just leave it blank. Thanks! Please move the Msg line into the body of the code block. Thank you!

Reply:
I made this fix.